sycl/test/abi/symbol_size.cpp

```diff
@@ -59,7 +59,7 @@ int main() {
   check_size<program, 16>();
   check_size<range<1>, 8>();
   check_size<sampler, 16>();
-  check_size<stream, 208>();
+  check_size<stream, 144>();
```

Comment:
I believe you'll have to update version of library now.

Reply:
Is there something besides the change already done in sycl/CMakeLists.txt?

Reply:
Please, refer to this document: https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md